### ISSUE TITLE
Incorrect name is defined for "Icon gallery" and "Icon dots" button.

### ISSUE THREAD

**Issue by billycamp** [Microsoft/EmbeddedSocial-iOS-SDK#702]
Category: Screen Reader
MAS Rule: MAS4.2.1
Repro Steps
1. Launch Embedded Social Test App.
2. Navigate to Hamburger button and double tap.
3. Navigate to Profile link and double tap.
4. Profile page will be appear.
5. Navigate  the page via swipe gesture.
6. Verify that name is defined correctly for Icon gallery and Icon dots button or not.
Expected:
Correct name should be defined for the Icon gallery and Icon dots button, so that screen reader announce correctly and user will get proper and correct information.
Actual:
Incorrect name is defined for the Icon gallery and Icon dots button.

Note: 
1. Same issue is repro for Close button, when open uploaded image in any topic.
2. Same issue is repro for Icon Hamburger button.
Test Environment:
OS: iOS 11.0.2
App: Embedded Social Test
App Version: 0.7.8 (8)
Tools: Voice Over, Keyboard
